item-test/test_refda_stdio.py

```diff
@@ -361,4 +361,51 @@ def test_odm(self):
         # items of the report
         self.assertEqual(1, len(rpt.items))
         self.assertEqual(6, rpt.items[0].value.size);
-        self.assertEqual(False, rpt.items[0].value[0][5].value); # Confirm rule is disabled
\ No newline at end of file
+        self.assertEqual(False, rpt.items[0].value[0][5].value); # Confirm rule is disabled
+
+
+    def test_odm_var_const(self):
+        self._start()
+
+        LOGGER.setLevel(logging.INFO)
+
+        self._send_execset(
+            'ari:/EXECSET/n=123;(//ietf/dtnma-agent/CTRL/ensure-odm(ietf, 1, !test-model-1, -1))'
+        )
+        rptset = self._wait_rptset().value
+
+        self._send_execset('ari:/EXECSET/n=123;(//ietf/dtnma-agent/CTRL/ensure-var(//ietf/!test-model-1,test-var,1,//ietf/amm-semtype/IDENT/type-use(/ARITYPE/int),/AC/(1)))', False)
+        rptset = self._wait_rptset().value
+
+        self._send_execset('ari:/EXECSET/n=123;(//ietf/dtnma-agent/CTRL/inspect(//ietf/dtnma-agent/EDD/var-list(false)))')
+        rptset = self._wait_rptset().value
+        rpt = rptset.reports[0]
+        self.assertEqual(1, len(rpt.items))
+        self.assertEqual(2, rpt.items[0].value.size)
```

Comment:
For the sake of test understanding, it can be good to add a `self.assertIsInstance` just before digging deeply into members-of-members. Not necessarily a test need but helpful for humans to understand what is being asserted.

In this case, asserting that `items[0].value` is a table(?) object.

Reply:
Good point, added checks for `ari.Table`.